l10n_ch_hr_payroll/models/hr_contract.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 #
-#  File: hr_contract.py
+#  File: models/hr_contract.py
 #  Module: l10n_ch_hr_payroll
 #
-#  Created by sge@open-net.ch
+#  Created by cyp@open-net.ch
```

Comment:
Was this intended or author was changed because of a bad copy paste? (just making sure)